vendor/vendor-android/base/src/main/kotlin/com/malinskiy/marathon/android/configuration/TimeoutConfiguration.kt

```diff
@@ -12,4 +12,5 @@ data class TimeoutConfiguration(
     var install: Duration = shell,
     var screenrecorder: Duration = Duration.ofMinutes(10),
     var screencapturer: Duration = Duration.ofMillis(300),
+    var socketTimeout: Duration = Duration.ofSeconds(30)
```

Comment:
@trevjonez Can we rename this to `socketIdleTimeout`? Sockets have more timeout parameters, ktor just doesn't expose it currently

Reply:
very valid point, done.